<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>402</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>724</width>
    <height>402</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>724</width>
    <height>402</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="DataTableWidget" name="pandastablewidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>363</width>
      <height>251</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>363</width>
      <height>251</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>363</width>
      <height>251</height>
     </size>
    </property>
    <widget class="QPushButton" name="pushButton_queding1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>20</width>
      <height>391</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>247</width>
      <height>78</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_logdatabase">
           <property name="text">
            <string>登录数据库</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_opendatabase">
           <property name="text">
            <string>打开数据库</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_openfile">
           <property name="text">
            <string>打开其他文件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_sqllanguage">
           <property name="text">
            <string>SQL语言</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_save">
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_datareturn">
           <property name="text">
            <string>数据还原</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_cleardata">
           <property name="text">
            <string>数据清空</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_view">
           <property name="text">
            <string>视图</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>数据归一化</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="radioButton_Minmax">
         <property name="text">
          <string>最小-最大规范化</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_Zscore">
         <property name="text">
          <string>z分数规范化</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_xiaoshudingbiao">
         <property name="text">
          <string>小数定标规范化</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox_attributes">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>60</width>
      <height>19</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择属性</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_comboboxattributes">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>属性值</string>
    </property>
   </widget>
   <widget class="DataTableWidget" name="qtpandas_attributewidget" native="true">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>270</width>
      <height>133</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>270</width>
      <height>133</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>270</width>
      <height>133</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>351</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_pciture"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>724</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPreprocess">
    <property name="title">
     <string>数据预处理</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuClassify">
    <property name="title">
     <string>分类</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuCluster">
    <property name="title">
     <string>聚类</string>
    </property>
   </widget>
   <addaction name="menuPreprocess"/>
   <addaction name="menuClassify"/>
   <addaction name="menuCluster"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataTableWidget</class>
   <extends>QWidget</extends>
   <header>qtpandas.views.DataTableView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
